testing/dart/BUILD.gn

```diff
@@ -52,8 +52,13 @@ foreach(test, tests) {
   flutter_frontend_server("compile_$test") {
     main_dart = test
     kernel_output = "$root_gen_dir/$test.dill"
-    extra_args = [ "-DkSkiaGoldWorkDirectory=$skia_gold_work_dir" ]
+    extra_args = [
+      "-DkFlutterBuildDirectory=$root_gen_dir",
```

Comment:
Use `rebase_path` on this like with the skia_gold_work_dir above so you get a non-gn-ified path.

Reply:
Done.